app/serializers/tag.js

```diff
@@ -0,0 +1,43 @@
+import DS from 'ember-data';
+
+export default DS.JSONSerializer.extend({
+
+  // Modify "response" objects to become "tag" objects.
+  // Only used for 'query' requests, such as: this.store.query('tag', { memberId: 4, chapterId: 102 }) .
+  normalizeQueryResponse(store, primaryModelClass, payload) {
+    var tags = [],
+      responses = payload.responses;
+
+    responses.forEach(function(response) {
+      tags.push({
+        type: 'tag',
+        id: parseInt(response.id),
+        attributes: {
+          chapterId: parseInt(response.surveyId),
+          questionId: parseInt(response.questions.questionId),
+          answer: response.questions.response,
+          questionNumber: -1,
+        },
+      });
+    });
+
+    return {
+      data: tags,
+    };
+  },
+
+  // Modify "tag" objects to become "response" objects.
+  // Only used for sending data, such as: tag.save() .
+  serialize: function(snapshot) {
+    var json = {
+      surveyId: snapshot.attr('chapterId'),
+      questions: {
+        questionId: snapshot.attr('questionId').toString(),
```

Comment:
`'questionId'` *must* be checked that it is a string or this will error. The API *should* return this property on the object, but it is not guaranteed.